source/lib/vagrant-openstack-provider/config/http.rb

```diff
@@ -0,0 +1,39 @@
+module VagrantPlugins
+  module Openstack
+    class HttpConfig
```

Comment:
This class is covered by tests (indirectly tested) but it would be great to have a tests for the `merge` method.